interfaces.go

```diff
@@ -23,6 +23,8 @@ type processorFinderFunc func(identity elemental.Identity) (Processor, error)
 
 type eventPusherFunc func(...*elemental.Event)
 
+type customHandlerFunc func() map[string]http.HandlerFunc
```

Comment:
I don't think this type is needed. If so the name is wrong. It's not a func

Reply:
It follows the pattern and yes its is needed. It's essentially the CustomHandlers that is passed around to functions without objects.